environment.yml

```diff
@@ -1,23 +1,21 @@
 name: echopype
 channels:
   - conda-forge
-  - defaults
 dependencies:
+  - python=3.8
+  - matplotlib
+  - netCDF4
+  - numpy
+  - pynmea2
+  - pytz
+  - xarray
+  - zarr
+  # extras
```

Comment:
Everything above is required and I kept the rest as "extras" b/c I'm not sure how this file is used. I usually don't avise people to add env files to the repo to avoid confusion. It is hard to keep everything in sync that way. Also, for source code repos I recommend to use the same req files with something like:

```shell
 conda create --name ECHOTYPE python=3.8 --file requirements.txt --file requirements-dev.txt
```

for example.

Reply:
I see, makes sense. Most below are for running the example notebooks. Some of them are not actually needed and can be deleted I think (e.g. click, pandas). I guess this is consistent with the suggestion that the notebooks should be separated out to another repo.

Reply:
Or this file belongs in the notebook folder. In the root folder it can be confused with a work/test/build env.

Reply:
Sounds good!